content/build/cache/backends/gha.md

```diff
@@ -99,6 +101,56 @@ For example:
     cache-to: type=gha,mode=max
 ```
 
+## Avoid GitHub Actions cache API throttling
+
+GitHub's [usage limits and eviction policy]https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy)
+causes stale cache entries to be removed after a certain period of time. By
+default, the `gha` cache backend uses the GitHub Actions cache API to check the
+status of cache entries.
+
+The GitHub Actions cache API is subject to rate limiting if you make too many
+requests in a short period of time, which may happen as a result of cache
+lookups during a build using the `gha` cache backend.
+
+```text
+#31 exporting to GitHub Actions Cache
+#31 preparing build cache for export
+#31 preparing build cache for export 600.3s done
+#31 ERROR: maximum timeout reached
+------
+ > exporting to GitHub Actions Cache:
+------
+ERROR: failed to solve: maximum timeout reached
+make: *** [Makefile:35: release] Error 1
+Error: Process completed with exit code 2.
+```
+
+To mitigate this issue, you can supply a GitHub token to BuildKit. This lets
+BuildKit utilize the standard GitHub API for checking cache keys, thereby
+reducing the number of requests made to the cache API.
+
+To provide a GitHub token, you can use the `ghtoken` parameter, and a
```

Comment:
```suggestion
To provide a GitHub token, you can use the `ghtoken` attribute, and a
```

I think we use "attribute" as wording

Reply:
I am happy to change it to attributes, but I mostly see it referred to as parameter

<img width="710" alt="image" src="https://github.com/docker/docs/assets/35727626/f7114d56-3c95-4f1e-98c2-589450af370c">

follow-up?

Reply:
Ah ok let's keep parameter for now then. We can check that in follow-up.